src/time.rs

```diff
@@ -0,0 +1,108 @@
+#![doc(cfg(feature = "time"))]
+
+//! Support for [::time] crate.
+use ::time::{Date, PrimitiveDateTime, OffsetDateTime, Time};
+use ::time::format_description;
+use leptos::*;
+use crate::*;
+
+#[derive(Default)]
+pub struct RenderTimeOptions {
+    /// Specifies a format string see [the time book](https://time-rs.github.io/book/api/format-description.html).
+    pub string: Option<String>,
+}
+
+/// Implementation for [`Date`] to work with the [`TableRow`] derive and the [`DefaultTableCellRenderer`]
+/// ``` 
+/// # use leptos_struct_table::*;
+/// # use leptos::*;
+/// # use ::time::Date;
+/// #[derive(TableRow, Clone)]
+/// #[table]
+/// struct SomeStruct {
+///     #[table(format(string = "[year]-[month]-[day]"))]
+///     my_field: Date
+/// }
+/// ```
+impl CellValue for Date {
+    type RenderOptions = RenderTimeOptions;
+    fn render_value(self, options: &Self::RenderOptions) -> impl IntoView {
+        if let Some(value) = options.string.as_ref() {
+            let format = format_description::parse(value).unwrap();
```

Comment:
I do think that this is a bit odd, that we unwrap here. I did not really know what else to do. 

From what I can tell Chrono panics internally on an invalid format like `"%0P"` and likewise this would panic in case you give it something like `"[anything"`

maybe it should be an expect or something instead. please let me know what you think on the matter!

Reply:
Yes let's do a descriptive expect here

Reply:
That was fast!